osu.Game/Audio/HitSampleInfo.cs

```diff
@@ -44,7 +45,7 @@ public class HitSampleInfo : ISampleInfo, IEquatable<HitSampleInfo>
         /// </summary>
         public int Volume { get; }
 
-        public HitSampleInfo(string name, string? bank = null, string? suffix = null, int volume = 0)
+        public HitSampleInfo(string name, string bank = SampleControlPoint.DEFAULT_BANK, string? suffix = null, int volume = 100)
```

Comment:
Now that the volume is 100 by default you also have to make sure that the legacy decoder properly overrides hit object volumes with volume from the control points. If the volume is 100 then it will not take the volume from the control points.

Reply:
Where's the code for this? Sounds flaky as hell, and untested.

Reply:
Okay turns out my concerns were misplaced. The `ConvertHitObjectParser` which creates the `LegacyHitSampleInfo` always assigns the volume of the `SampleBankInfo` whose volume is still 0 by default, so it works out just fine. For a moment I thought there were occurances of `LegacyHitSampleInfo` being created while decoding without a volume argument given.

Reply:
And even if that wasn't the case, `ConvertHitObjectParser`  only constructs `LegacyHitSampleInfo`s, and `LegacyHitSampleInfo` ctor still has 0 volume as a default, which makes the change of the default value in the base class largely irrelevant.